sass-style-guide.md

```diff
@@ -165,90 +172,67 @@ ul
   list-style-type: circle
 ```
 
-### Avoid element selectors
+#### Vermeide Elementselektoren
 
-Element selectors are expensive. Like the rule above, be more specific. Try using a class instead. Furthermore elements like `<div />` and `<span />` should always have a class-attribute in your markup because otherwise they have no function anyway and you can waive them.
+Elementselektoren sind schlecht (siehe Regel darüber) und erhöhen die Spezifität unnötigerweise. Auch hier sollte auf eine Klasse zurückgegriffen werden. Außerdem sollten Elemente wie `<div>` und `<span>` im Markup immer ein Klassenattribut besitzen. Andernfalls wären sie sowieso überflüssig.
 
-**bad**
+**schlecht**
 
 ``` sass
 .navigation div
 .navigation span
 .navigation ul
 ```
 
-**good**
+**gut**
 
 ``` sass
 .navigation--box
 .navigation--title
 .navigation--list
 ```
 
-### Avoid qualifying class names with element selectors
+#### Vermeide eine Überspezifizierung durch Elementselektoren
 
-It’s counterproductive because you unnecessary heighten the specifity.
+Eine Überspezifizierung durch Elementselektoren hat zur Folge, dass der CSS-Code ans Markup gekoppelt wird. Würde man das Element im Markup ändern, z.B. aus einer `h2` eine `h3` machen, müsste man dies auch im Stylesheet tun, was einen erhöhten Wartungsaufwand nach sich zieht. Obendrein wird die Spezifität unnötigerweise erhöht.
 
-**bad**
+**schlecht**
 
 ``` sass
 ul.link-list
 div.example
 a.back
 ```
 
-**good**
+**gut**
 
 ``` sass
 .link-list
 .example
 .back
 ```
 
-### Avoid the descendant selector
+#### Vermeide Nachfahrenselektoren
 
-The descendant selector is the most expensive selector in CSS. You should target directly if possible.
+Der Nachfahrenselektor ist der teuerste Selektor in CSS. Gib dem entsprechenden Element besser eine Klasse und style es direkt.
 
-**bad**
+**schlecht**
 
 ``` sass
 .link-list li a
 ```
 
-**good**
+**gut**
 
 ``` sass
 .link-list--link
 ```
 
-### Avoid deep nesting
-
-Following to the rule above you should also try to nest your selectors maximum 3 levels deep (pseudo-elements and pseudo-classes not counting).
-
-**bad**
-
-``` sass
-.link-list
-  li
-    a
-      span
-        &:before
-          content: 'i'
-```
-
-**good**
+#### Vermeide den selben Selektor fürs Styling (CSS) und fürs Verhalten (JS) zu verwenden
```

Comment:
Wenn man auf _Single File Components_ bei vue.js zurückgreift, hat man auch alles an einem Ort, was es definitiv einfacher macht. Da finde ich es auch sinnvoll! Splitte ich jedoch meinen Code in HTML-, CSS- und JS-Dateien auf, so wie wir es derzeit tun, halte ich den oben beschriebenen Ansatz für gangbarer.